### PR TITLE
LPS-92649 Update themes package versions

### PR DESCRIPTION
**Pull request by marcoscv-work**

### CODE REVIEW

File: modules/apps/frontend-theme-westeros-bank/frontend-theme-westeros-bank/package.json [hunk 1]
```diff
@@ -4,9 +4,12 @@
 		"url": "https://issues.liferay.com/"
 	},
 	"devDependencies": {
-		"gulp": "^3.8.10",
-		"liferay-theme-deps-7.1": "8.0.0-rc.2",
-		"liferay-theme-tasks": "8.0.0-rc.2"
+		"gulp": "3.9.1",
+		"liferay-theme-tasks": "^9.0.0-alpha.1",
+		"compass-mixins": "0.12.10",
+		"liferay-frontend-common-css": "^1.0.4",
```
Comment (jbalsas) [inline modules/apps/frontend-theme-westeros-bank/frontend-theme-westeros-bank/package.json]: Didn't we release a nightly of this too, @wincent?

Comment (wincent) [reply modules/apps/frontend-theme-westeros-bank/frontend-theme-westeros-bank/package.json]: No. IIRC, didn't that one blow up when you tried to release it? It is definitely not listed on the "Versions" tab here: https://www.npmjs.com/package/liferay-frontend-common-css

Comment (jbalsas) [reply modules/apps/frontend-theme-westeros-bank/frontend-theme-westeros-bank/package.json]: 🤔

Comment (jbalsas) [reply modules/apps/frontend-theme-westeros-bank/frontend-theme-westeros-bank/package.json]: If that's how you remember it, that's probably how it happened :D

**Comment by jbalsas** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/70559. [See changes here.](https://github.com/jbalsas/liferay-portal/compare/b77cd09e78e0~1...b77cd09e78e0) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by jbalsas** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/70562. [See changes here.](https://github.com/jbalsas/liferay-portal/compare/76e19d701482~1...76e19d701482) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by jbalsas** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/70565. [See changes here.](https://github.com/jbalsas/liferay-portal/compare/98af592f596b~1...98af592f596b) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by jbalsas** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/70629. [See changes here.](https://github.com/jbalsas/liferay-portal/compare/6b66d382a4b1~1...6b66d382a4b1) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**